db/seeds.rb

```diff
@@ -34,225 +106,207 @@
-puts '--------------------------------------------------------------------'.light_black
+puts ''
 
 #############################################################################
 #---------------------------SEED DB WITH DIETS------------------------------#
 #############################################################################
+puts '-----------------------------Diets----------------------------------'.light_black
 
 diet_counter = 0
 @diets_list.length.times do
+  next if Diet.count == 5
+
   diet = Diet.create!(name: @diets_list[diet_counter])
   diet_counter += 1
   print "#{diet.id}. "
   puts diet.name.light_blue
 end
 puts "#{'✓ Diets '.light_green}created"
-puts '--------------------------------------------------------------------'.light_black
+puts ''
 
 #############################################################################
-#-----------------------SEED DB WITH INGREDIENT ITEMS-----------------------#
-#-------!These items do not have any allergens or dietary restrictions!-----#
+#----------------------------DEMO PERSONAS-----------------------------------#
 #############################################################################
+puts '----------------------------Personas--------------------------------'.light_black
 
-# Creates users with ingredient items to fill DB.
-user = User.create!(
-  email: Faker::Internet.email,
-  username: Faker::Name.first_name + Faker::Creature::Dog.name,
+# Demo user Justin (starts as 'RECEIVER' then becomes a 'GIVER' by the end of pitch).
+@justin = User.create!(
+  email: 'justin@foodfor.all',
+  username: 'Justin',
   password: '123456',
-  address: locations.sample
+  address: '5333 Av. Casgrain, montreal'
+)
+puts " Demo persona: #{@justin.username.light_cyan} has been created."
+puts ''
+# Demo user Shayna will be the GIVER that justin receives a meal from.
+@shayna = User.create!(
+  email: 'shayna@foodfor.all',
+  username: 'Shayna',
+  password: '123456',
+  address: '5057 rue de bullion, montreal'
+)
+shaynas_meal = Item.create!(
+  user_id: @shayna.id,
+  name: @meals.last,
+  status: 'available',
+  item_type: 'meal',
+  description: @meal_descriptions.last,
+  expiration_date: Faker::Date.between(from: 2.days.from_now, to: 5.days.from_now)
 )
-puts "#{'✓'.light_green} Demo user: #{user.username.light_cyan} has been created."
 
-5.times do
-  ingredient = Item.create!(
-    user_id: user.id,
-    name: Faker::Food.vegetables,
-    status: @status_list.sample,
-    item_type: 'ingredient',
-    description: Faker::Food.description,
-    expiration_date: Faker::Date.between(from: 2.days.from_now, to: 5.days.from_now)
-  )
-end
-puts "#{'5'.blue} ingredient items created for #{user.username.light_cyan} "
-puts '--------------------------------------------------------------------'.light_black
+puts " Demo persona: #{@shayna.username.light_cyan} has been created with a #{shaynas_meal.name.cyan} meal"
+add_allergens_and_diets(shaynas_meal)
```

Comment:
Method from above

Reply:
Great way to demonstrate teamwork during final demo - speaker: Denzel, driver: Marlene, receiver: Williams and giver: Shayna. All team members will present in a way!